### ISSUE TITLE
Possible issue with chest generation

### ISSUE THREAD

**Issue by Teddy-Maz** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#72]
**Describe the bug**
I've been cobbling a pack together for personal use and I've found that a handful of village chests don't generate chest loot when spawning. It's hard to say exactly what causes it because I have a lot of mods going, but my best guess is one or more of the Artifacts mod, Enigmatic Legacy mod, or Relics mod. Each one injects their own loot into chests all over the world, which I think is causing some chests to generate completely empty. This has been tested with samples of CTOV villages of every size and in a handful of different biome types. Certain chests (cartographers, clerics if they have them, farmers) seem to generate with little to no issue, making the bug hard to reproduce.

**To Reproduce**
Steps to reproduce the behaviour:
1, Install CTOV and any one of the following mods
2. use /locate to search up villages of various sizes and biome types available in the mod
3. Open a handful of chests across the village; Barrels seem to be unaffected, and certain chests generate with no issue. Others (more than half in my tests) will not generate any loot at all. 

**Expected behaviour**
The mod description suggests that the chests in these villages use standard villager loot tables, and thus should generate without issue. As far as I'm aware, the mods listed above aren't meant to cause such issues when populating their items into chests.

**Minecraft Version**
1.18.2 (Forge 40.2.9)

**CTOV Version**
2.9.4

**Add-ons used**
Too many to list! Most importantly, CTOV plus a handful of loot-based mods

**EDIT: Logs**
[debug.log](https://github.com/ChoiceTheorem/ChoiceTheorem-s-overhauled-village/files/12587640/debug.log)
[latest.log](https://github.com/ChoiceTheorem/ChoiceTheorem-s-overhauled-village/files/12587656/latest.log)

**Comment by ChoiceTheorem** [comment]
This type of issue is exactly the one I'd prefer to have a log with. You'll pinpoint the issue with it.

**Comment by Teddy-Maz** [comment]
> This type of issue is exactly the one I'd prefer to have a log with. You'll pinpoint the issue with it.

I can provide a log, is there something specific to look for when reading it over?

**Comment by ChoiceTheorem** [comment]
Invalid chest loot ctov:[something I goofed up]

**Comment by Teddy-Maz** [comment]
> Invalid chest loot ctov:[something I goofed up]

Done and done! I loaded up a new survival world, fortunately spawned right next to one of your villages, went into the Blacksmith's and opened up an empty chest. I read the log over and didn't see any warnings or errors that called out CTOV specifically, but I also don't have the eye for those kinds of details. I gave you latest and debug just in case.
